manifest-lock.overrides.aarch64.yaml

```
@@ -0,0 +1,3 @@
+packages:
+  libgcc:
+    evra: 11.2.1-7.fc36.aarch64
```

```suggestion
    evra: 11.2.1-7.fc36.aarch64
    metadata:  
      reason: https://
      type: pin
```

Add the metadata and type.